frontend/src/concepts/k8s/pods/usePodContainerState.ts

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { PodContainer } from '~/types';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import usePod from '~/concepts/k8s/pods/usePod';
+import { getPodContainers } from '~/concepts/k8s/pods/utils';
+import { PodKind } from '~/k8sTypes';
+
+const usePodContainerState = (
+  podName: string,
+): {
+  pod: PodKind | null;
+  podLoaded: boolean;
+  podError: Error | undefined;
+  podContainers: PodContainer[];
+  selectedContainer: PodContainer | null;
+  setSelectedContainer: (podContainer: PodContainer | null) => void;
```

Comment:
@christianvogt not sure I follow -- can you clarify what you mean by `arbitrary`? 

I _think_ you're saying it was added to this hook for nothing ... but that doesn't make sense because when the pulled containers and pod changes we want to throw away the selected container. Which is both hooks right under the usage. So what am I missing?

Reply:
Seems like it was an arbitrary choice to house the selected container state here in this hook. But it would be better suited to be owned by the logs UI since its the UI that owns and manipulates selection.

Reply:
We talked -- it's in the wrong spot... moving & renaming. My bad!